kinto/core/initialization.py

```diff
@@ -322,7 +340,7 @@ def install_middlewares(app, settings):
 
 def setup_logging(config):
     """Setup structured logging, and emit `request.summary` event on each
-    request, as recommanded by Mozilla Services standard:
+    request, as recommended by Mozilla Services standard:
```

Comment:
Good catch, do you mind filling a pull-request for this one change only?

Reply:
No problem  [PR #1455](https://github.com/Kinto/kinto/pull/1455)